packages/database/lib/schema.ts

```diff
@@ -41,23 +54,34 @@ function pgEnum<T extends string>(columnName: string, enumObj: Record<any, T>) {
 
 export const daos = pgTable('daos', {
   daoAcct: pubkey('dao_acct').primaryKey(),
-  name: varchar('name').notNull(),
-  url: varchar('url').notNull(),
+  name: varchar('name'),
```

Comment:
need to add programAcct (the Autocrat Program), through that we could also add, version...